src/Components/Shared/Header/Header.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+export default class header extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {};
```

Comment:
When we are not tracking any State in our Component.... We use Functional Components. Thus, You should convert this one into a Functional one.